tests/test_project_script.py

```
@@ -407,3 +407,389 @@ def test_python_3_10_proceeds(self, mock_project_dir, capsys):
+        # pyproject.toml
+        (tmp_path / "pyproject.toml").write_text(
+            "# Project configuration for Python projects"
+            " using the Agentive Starter Kit\n"
+            '[build-system]\nrequires = ["setuptools>=61.0"]\n'
+        )
+
+        # tests/conftest.py
+        tests_dir = tmp_path / "tests"
+        tests_dir.mkdir()
+        (tests_dir / "conftest.py").write_text(
+            '"""\nShared fixtures for the' " agentive-starter-kit test suite.\n" '"""\n'
```

Ruff (0.15.2)

[warning] Implicitly concatenated string literals on one line

(ISC001)